docs/guides/cloud/integrations/slack.mdx

```diff
@@ -1,228 +1,143 @@
 
-<DocsImage
-  src="/img/guides/cloud/integrations/slack/cypress-slack-integration-channel-feed.jpg"
-  alt="Cypress notification feed in Slack channel"
-  width={500}
-/>
+The Cypress app for Slack allows you to:
 
-## Install the Cypress integration for Slack
+- Improve cross-team collaboration by instantly surfacing Cypress results to the teams that need visibility
+- Confirm that key tests pass prior to launching new products or features
+- Reduce the time it takes to catch failed tests
 
-:::caution
+<DocsVideo
+  src="https://www.youtube.com/embed/hnDPz7dognY?si=PRRy4f5n6f1jzfVJ"
+  title="Cypress app for Slack"
+/>
 
-<strong>Ownership Requirements</strong>
+## Getting started
 
-In order to install the Cypress integration for Slack, you must be an admin or
-owner of both your Cypress Cloud organization and your Slack workspace.
+You need just three things to get started:
 
-:::
+- Cypress Cloud account
+- Ability to configure Cypress Cloud integrations - you must be an admin or the owner of your Cypress Cloud organization.
+- Ability to authorize Slack apps - by default, Slack allows any workspace member to authorize apps, however workspace owners may require app approvals before installation. See this [Slack help article](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace) for more details.
 
-**To install the Cypress integration for Slack:**
+## Install the Cypress app for Slack
 
-1. Go to Cypress Cloud
-   [Organizations page](https://cloud.cypress.io/organizations) or open the
-   organization switcher.
-2. Select the organization you wish to integrate with Slack.
+1. In the Cypress Cloud, visit the Integrations page for the organization you wish to integrate with Slack.
+2. Click the **Enable** button in the Slack section.
+3. A popup window will be displayed requesting permission for Cypress to access the
+   workspace.
+4. After you approve the app permissions, you can choose your Slack workspace and channel to
+   associate with the installation.
+5. The installation is complete!
 
-   <DocsImage
-     src="/img/guides/cloud/organizations/select-cypress-organization.jpg"
-     alt="Select an organization"
-     width={500}
-   />
+Now Cypress will post run results for all projects in your organization to the specified Slack channel.
 
-3. Visit the selected organization's **Integrations** page via the side
-   navigation.
+## Slack configuration
 
-   <DocsImage
-     src="/img/guides/cloud/organizations/navigate-to-organization-integrations.jpg"
-     alt="Install Cypress Slack from Integrations"
-     width={500}
-   />
+The Slack messages can be tailored to match your teams' needs.
 
-4. Click the **Enable** button in the Slack section.
+- The Slack app can send results to one or more public or private Slack channels or directly to a specific user
```

Comment:
Is 'Slack app' right here?

Reply:
Slack app came from Slack's documentation: https://api.slack.com/docs/apps